replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/Syncer.java

```diff
@@ -185,6 +187,45 @@ SyncResponse sync() {
       return new SyncResponse(replicationStatus.getStatus());
     }
 
+    Optional<ReplicationItem> getReplicationItem(String metadataId) {
+
+      ReplicationItem item =
+          replicationItemManager.getItem(metadataId, sourceName, destinationName).orElse(null);
+      ReplicationItem reverseDir =
+          replicationItemManager.getItem(metadataId, destinationName, sourceName).orElse(null);
+      if (item == null && reverseDir == null) {
+        return Optional.empty();
+      }
+      if (item != null
```

Comment:
wouldn't worry about it for now, but seems like you could check for `item != null` for the rest of this block and not check for it multiple times.